### PR TITLE
Adding the option to insert a social profile link

### PR DESCRIPTION
**Pull request by OzzyCzech**
### Request type

- [x] Feature
- [ ] Fix
- [ ] Refactor
- [x] Tests
- [x] Documentation

### Summary

There are [several ways](https://alessandrorossini.org/the-sad-story-of-the-vcard-format-and-its-lack-of-interoperability/) to solve social networks profile on VCARD format a most clean one is use URL, but widely used is also property `X-SOCIALPROFILE` to VCard allow add different social networks like Twitter, Facebook and others:

```text
X-SOCIALPROFILE;type=twitter;x-user=twitteruser:http://twitter.com/twitteruser
X-SOCIALPROFILE;type=facebook;x-user=facebookuser:http://www.facebook.com/facebookuser
X-SOCIALPROFILE;type=flickr;x-user=flickruser:http://www.flickr.com/photos/flickruser
X-SOCIALPROFILE;type=linkedin;x-user=linkedinuser:http://www.linkedin.com/in/linkedinuser
X-SOCIALPROFILE;type=myspace;x-user=myspaceuser:http://www.myspace.com/myspaceuser
X-SOCIALPROFILE;type=sinaweibo;x-user=sinauser:http://weibo.com/n/sinauser
X-SOCIALPROFILE;type=Customsocial;x-user=custom socialuser:x-apple:custom%20socialuser
```

### Change description

Add method `public addSocial(url: string, type: string, user: string = '')` to the VCard object and update readme and tests.

### Check lists

- [x] Tests passed
- [x] Coding style respected

### CODE REVIEW

**Comment by OzzyCzech** [comment]
All changes from #37 are here, I am not able to reopen that issue again, sorry for messing up

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-name: "CodeQL"
+name: 'CodeQL'
```
Comment (joaocarmo) [inline .github/workflows/codeql.yml:1]: **issue:** can we remove the changes to this file, since it's not part of the suggested changes?

**Review by joaocarmo — CHANGES_REQUESTED** [review]
LGTM!

Just one minor issue with the changes to the GitHub workflows definitions.

File: .github/workflows/codeql.yml (hunk 1 above)
Comment (OzzyCzech) [reply .github/workflows/codeql.yml]: fixed

**Review by joaocarmo — APPROVED** [review]

**Pull request merged**